csrc/torch_bindings.cpp

```diff
@@ -135,6 +135,10 @@ TORCH_LIBRARY_EXPAND(TORCH_EXTENSION_NAME, ops) {
 
   // CUTLASS w8a8 GEMM, supporting symmetric per-tensor or per-row/column
   // quantization.
+  ops.def("cutlass_scaled_mm_supports_fp8", &cutlass_scaled_mm_supports_fp8);
+  ops.impl("cutlass_scaled_mm_supports_fp8", torch::kCUDA,
+           &cutlass_scaled_mm_supports_fp8);
```

Comment:
The above comments should be for `cutlass_scaled_mm`, and we should have a comment for this op, such as `check whether cutlass scaled mm fp8 is supported in the current env`.